packages/@okta/vuepress-site/docs/guides/manage-orgs-okta-aerial/main/index.md

```diff
@@ -0,0 +1,377 @@
+
+```bash
+Authorization: Bearer ${access_token}
+
+{
+  "domain": "${yourOktaDomain}",
+  "cell": "OK1"
+}
+```
+
+### Response example
+
+<!-- We have this note in the Google doc. What to do?
+
+Note: this API is still in development. We will update the documentation with more accurate examples as we progress in development. -->
+
+The id of this record is the `orgId` to use in the URL for enabling Products:
+
+```json
+{
+  "accountId": "string",
+  "name": "string",
+  "region": "string",
+  "cell": "string",
+  "domain": "string",
+  "status": "string",
+  "createdDate": "string",
+  "createdBy": "string",
+  "id": "string"
+}
+```
+
+## Enable products in the Org
+
+<!-- should add a bit of concept content here -->
+
+<ApiOperation method="put" url="https://aerial-{region}/{accountId}/api/v1/orgs/{orgId}/products" />
+
+### Request and response example
+
+```json
+{
+  "accountId": "string",
+  "name": "string",
+  "region": "string",
+  "cell": "string",
+  "domain": "string",
+  "status": "string",
+  "createdDate": "string",
+  "createdBy": "string",
+  "id": "string"
+}
+```
+
+## Configure the Org
+
+To pre-configure groups, apps, and policies in each org, the Aerial API client needs the following:
+
+- The domain of the org
+- The API token returned by the Org creator API
+
+#### Request example
+
+```bash
+curl --location --request POST 'https://${newOrgDomain}/api/v1/apps \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "name": "bookmark",
+  "label": "Sample Bookmark App",
+  "signOnMode": "BOOKMARK",
+  "settings": {
+    "app": {
+      "requestIntegration": false,
+      "url": "https://example.com/bookmark.htm"
+    }
+  }
+}'
+```
+
+## Manage an Org's status
+
+<ApiOperation method="put" url="https://aerial-{region}/{accountId}/api/v1/orgs/{orgId}/status" />
+
+Deactivate an org by calling the `/status` endpoint. Deactivated orgs don’t count toward billing. Users in the org can’t use Okta’s services or sign-in to Okta.
+
+<div class="three-quarter">
+
+![Deactivate an Org](/img/aerial-change-status.png)
+
+</div>
+
+### Request example
+
+```bash
+{
+   "status": "INACTIVE"
+}
+```
+
+### Response example
+
+```bash
+{
+  "id": "00o133mJ27DoonJz50g4",
+  "status": "INACTIVE",
+  ...
+}
+```
+
+## List the configured Products
+
+<ApiOperation method="get" url="https://aerial-{region}/{accountId}/api/v1/orgs/{orgId}/products" />
+
+### Response example
+
+```bash
+[
+  {
+    "id": "P000052",
+    "name": "IT Products - SSO"
+  },
+  {
+    "id": "P000131",
+    "name": "IT Products - MFA"
+  }
+]
+```
+
+## Enable a Product in an Org
+
+<ApiOperation method="put" url="https://aerial-{region}/{accountId}/api/v1/orgs/{orgId}/products" />
+
+Enables a product with ID P000139 in the org (IT Products - Lifecycle Management with 1 OIN App)
+
+> **Note:** The `name` property is ignored, and can be included to simplify the client implementation.
```

Comment:
Please LMK if I changed the meaning of this with my edit
```suggestion
> **Note:** Since Okta ignores the `name` property, you can include it to simplify the client implementation.
```